bluesky/tests/test_est_time.py

```diff
@@ -0,0 +1,105 @@
+from bluesky.simulators import est_time
+from ophyd.sim import hw
+from bluesky.run_engine import RunEngine
+from ophyd.telemetry import TelemetryUI
+from bluesky.plans import scan, count
+from bluesky.plan_stubs import mv, sleep
+import pytest
+
+#setup some testing objects
+RE=RunEngine()
```

Comment:
Instead of sharing instances across the module, make a fresh one per test by using pytest fixtures. This avoids unexpected and hard-to-debug interactions between separate tests.

Reply:
see above